# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: layout/base/nsFrameTraversal.h
# Commit: b3f0e5c7874d
# Full Hash: b3f0e5c7874dbd470d96e476e116740e4ca35925
# Author: David Zbarsky <dzbarsky@gmail.com>
# Date: 2012-08-06 14:40:43
# Regressor Bug: 779684
# File Overlap Count: 1
# Description:
#   Bug 779684: Create a stack-based nsFrameIterator class out of nsIFrameTraversal and nsIFrameEnumerator r=roc
# ==============================================================================

diff -r 4bc7154190dd -r b3f0e5c7874d layout/base/nsFrameTraversal.h
--- a/layout/base/nsFrameTraversal.h	Sun Aug 05 22:55:18 2012 -0400
+++ /dev/null	Thu Jan 01 00:00:00 1970 +0000
@@ -1,38 +0,0 @@
-/* -*- Mode: C++; tab-width: 2; indent-tabs-mode: nil; c-basic-offset: 2 -*- */
-/* This Source Code Form is subject to the terms of the Mozilla Public
- * License, v. 2.0. If a copy of the MPL was not distributed with this
- * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
-#ifndef NSFRAMETRAVERSAL_H
-#define NSFRAMETRAVERSAL_H
-
-#include "nsIFrame.h"
-#include "nsIFrameTraversal.h"
-
-nsresult NS_NewFrameTraversal(nsIFrameEnumerator **aEnumerator,
-                              nsPresContext* aPresContext,
-                              nsIFrame *aStart,
-                              nsIteratorType aType,
-                              bool aVisual,
-                              bool aLockInScrollView,
-                              bool aFollowOOFs);
-
-nsresult NS_CreateFrameTraversal(nsIFrameTraversal** aResult);
-
-class nsFrameTraversal : public nsIFrameTraversal
-{
-public:
-  nsFrameTraversal();
-  virtual ~nsFrameTraversal();
-
-  NS_DECL_ISUPPORTS
-
-  NS_IMETHOD NewFrameTraversal(nsIFrameEnumerator **aEnumerator,
-                               nsPresContext* aPresContext,
-                               nsIFrame *aStart,
-                               PRInt32 aType,
-                               bool aVisual,
-                               bool aLockInScrollView,
-                               bool aFollowOOFs);
-};
-
-#endif //NSFRAMETRAVERSAL_H